utils/validation/fields.ts

```diff
@@ -0,0 +1,13 @@
+export function testPhoneNumber(phonenumber: string) {
```

Comment:
Was there any particular reason why the [`validator` library ](https://www.npmjs.com/package/validator)wasn't used?

Here's a code snippet where we used it before:
https://github.com/uwblueprint/social-diversity-for-children/blob/dcdacb840ecb4de36370aa7490256dc23336156b/utils/validation/user.ts#L90-L100

Reply:
If you feel that it isn't able to work, perhaps we should do our own regex validation in the code snippet too then instead of using `validator`

Reply:
@GodGreg I feel like we should definitely use `validator` instead, especially if we want to make consistent use of the existing validation utils.

Reply:
I was unaware this existed; I will make the change